### PR TITLE
Make GDI+ private types opaque

### PR DESCRIPTION
**Pull request by riverar**
Fixes: #1821

Updates GDI+ private classes in metadata from `NativeTypeDef`s with a single pointer-sized field to opaque empty structs for better accuracy and proper handling.

### CODE REVIEW

**Comment by riverar** [comment]
@AArnott If you'd like to run this through CsWin32, here's a winmd:
[artifact.zip](https://github.com/microsoft/win32metadata/files/15270556/artifact.zip)

**Review by mikebattista — APPROVED** [review]

**Comment by mikebattista** [comment]
Will merge pending approval from @AArnott.

File: generation/WinSDK/manual/GdiPlus.cs [hunk 1]
```diff
@@ -0,0 +1,31 @@
+using System;
+using System.Runtime.InteropServices;
+using Windows.Win32.Foundation.Metadata;
+
+namespace Windows.Win32.Graphics.GdiPlus
+{
+    public struct GpGraphics;
+    public struct GpBrush;
+    public struct GpTexture
```
Comment (riverar) [inline generation/WinSDK/manual/GdiPlus.cs:9]: The missing semi-colons here are interesting, will fix that up.

**Comment by AArnott** [comment]
I tested CsWin32 with this winmd, and full generation produces compilable code. I sign off.

**Review by AArnott — APPROVED** [review]

**Review by mikebattista — APPROVED** [review]

**Pull request merged**